e2e-tests/tfm/cypress/fixtures/create-tfm-deal-to-insert-into-db.js

```diff
@@ -393,4 +394,4 @@ const createTfmDealToInsertIntoDb = (ukefDealId, companyName, dealObjectId) => (
   },
 });
 
-module.exports = createTfmDealToInsertIntoDb;
+export default  createTfmDealToInsertIntoDb;
```

Comment:
linting - extra space here

Reply:
I think theres a few linting issues - I wont comment on all of them but you probably need to run `npm run lint:all` in the root folder.
Out of interest did you skip the commit hook?

Reply:
I was naughty and skipped it once - all linting is resolved now